waiter/src/waiter/token.clj

```diff
@@ -157,6 +184,12 @@
                   token-set (->> tokens (map :token) set)]
               (kv/store kv-store owner-key token-set))))))))
 
+(defn compute-last-modified-etag
```

Comment:
This function doesn't compute anything and the comment doesn't describe what it does.  Please just inline.